### DISCUSSION TITLE
Include custom css classes

### DISCUSSION THREAD

**Discussion by raissa-amaral** [danielfrg/jupyter-flex#119] [Ideas]
Hello!

First of all, thanks to everyone creating/maintaining this library - it is really useful to my current needs. I do have a suggestion/question though..

Is it possible to add custom css classes to the jupyter cells? Similar to what is available for https://mui.com/\but for any custom class we wish to create.

For instance, let's say I need to hide a specific cell from my dashboard, but I need it to be in the jupyter notebook, I could create a css class named "hidden" and assign it to the tag:
![image](https://user-images.githubusercontent.com/97529725/158771962-4747c513-510c-4fb9-9f6b-e751ce25d624.png)

I've tried a few combinations but it doesn't look like this is possible at the moment, or if it is, what is the currect usage/syntax?

Thanks a lot,
Raissa

**Comment by danielfrg** [comment]
Hi, thanks for the comment.

For your specific use case you don't need to hide any cells, all the cells will be executed and only those selected with other tags like `body` will be shown.

You can also add classes like you mentioned with `class=value` but that only makes sense if you combine it with `body`, with out the `body` tag it will not be shown in the dashboard. You can see more info here: https://jupyter-flex.danielfrg.com/customization/

**Reply by raissa-amaral** [reply]
Thanks!
